app/src/main/java/com/asfoundation/wallet/ui/iab/InAppPurchaseInteractor.java

```diff
@@ -205,9 +205,11 @@ public Single<Boolean> isWalletFromBds(String packageName, String wallet) {
     if (packageName == null) {
       return Single.just(false);
     }
-    return bdsInAppPurchaseInteractor.getWallet(packageName)
-        .map(wallet::equalsIgnoreCase)
-        .onErrorReturn(throwable -> false);
+    return Single.just(true);
```

Comment:
it is always true, is it real or even necessary? and why to keep the old logic commented?